setup.py

```diff
@@ -83,11 +111,9 @@ def read(fname):
     entry_points={'console_scripts': [
         'django-admin = django.core.management:execute_from_command_line',
     ]},
-    install_requires=['pytz', 'sqlparse', 'asgiref'],
-    extras_require={
-        "bcrypt": ["bcrypt"],
-        "argon2": ["argon2-cffi >= 16.1.0"],
-    },
+    install_requires=['asgiref', 'pytz', 'sqlparse'],
+    extras_require=extras_require,
+    tests_require=tests_require,
```

Comment:
`tests_require` is deprecated in [setuptools](https://setuptools.readthedocs.io/en/latest/setuptools.html#new-and-changed-setup-keywords) since [41.5.0](https://setuptools.readthedocs.io/en/latest/history.html#v41-5-0) (27 Oct 2019). (It is also being unused by Django anyway.)